server/controllers/chartsController/index.js

```diff
@@ -54,7 +56,12 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
 
       labelMap.set(label, { name: label, color })
       processData({ shouldCountSubject, dataMap, dataKey, totalsDataKey })
-      processTotals({ shouldCountSubject, studyTotals, study, targetValue })
+      processTotals({
+        shouldCountSubject,
+        studyTotals,
+        study: siteName,
```

Comment:
should we rename `study` to `siteName`?

Reply:
thanks for this
https://github.com/AMP-SCZ/dpdash/pull/246/commits/9d9d1b15c3134f5ae991e11553bcddefde75cb55